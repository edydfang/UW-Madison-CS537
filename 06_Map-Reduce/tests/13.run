./sort tests/1.txt tests/2.txt tests/3.txt 3 1 4
