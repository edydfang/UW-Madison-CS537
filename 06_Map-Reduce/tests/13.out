3
3456346
34565363
523654
653465
75687785
3333333333
3333333334
3344556677
